### PR TITLE
v10 layout

### PR DESCRIPTION
**Pull request by tristen**
Broad stroke layout changes without new functionality.

### CODE REVIEW

**Comment by tristen** [comment]
@AlexanderBelokon I know this PR is on the big size. Some of that has to do with the differences to dependencies but would 🙏 a review from you.

File: .eslintrc [hunk 1]
```diff
@@ -34,21 +35,8 @@
...
     "react/require-render-return": ["error"],
-    "react/style-prop-object": ["error"],
-  },
-  "overrides": [
-    {
-      "files": [
-        "**/*.ts?(x)"
-      ],
-      "rules": {
-        "no-unused-vars": [
```
Comment (AlexanderBelokon) [inline .eslintrc:48]: This one is super obnoxious, good riddance)

**Review by AlexanderBelokon — APPROVED** [review]
It's perfect! ❤️

I will miss per-channel xy-pickers though)

Tiny nit: can we just have a space there if there's no alpha channel?:

https://user-images.githubusercontent.com/2198527/217272182-a92b16ee-80ec-4738-9942-7d9de24b8cbc.mp4

And a tiny personal preference: I like HSV canvas much more than the HSL canvas, it's much easier to pick a deep, saturated color and there's no reason (in my opinion) to have so much space dedicated to pure white)

Since our xy-picker's colorspace is no longer associated with the currently selected representation, we can choose anything for the picker now, right?

**Comment by tristen** [comment]
> And a tiny personal preference: I like HSV canvas much more than the HSL canvas, it's much easier to pick a deep, saturated color and there's no reason (in my opinion) to have so much space dedicated to pure white)

You raise a good point. Let's debate it in a separate ticket.

**Comment by tristen** [comment]
> Tiny nit: can we just have a space there if there's no alpha channel?:

Why the space? I'm not sure I agree with doing that : P

**Comment by AlexanderBelokon** [comment]
Well otherwise the dropdown arrow is jumping around, moreover, it jumps around even for HEX that doesn't support alpha!

**Pull request merged**